packages/react-runtime-dev-new/react.js

```diff
@@ -0,0 +1,17 @@
+import { Meteor } from 'meteor/meteor';
+
+import { checkNpmVersions } from 'meteor/tmeasday:check-npm-versions';
+checkNpmVersions({
+  'react': '0.14.x',
+  'react-dom': '0.14.x'
+});
+
+const ReactDev = require('react');
+const ReactDOMDev = require('react-dom');
+
+let ReactDOMServerDev;
+if (Meteor.isServer) {
+  ReactDOMServerDev = require('react-dom/server');
+}
+
+export { ReactDev, ReactDOMDev, ReactDOMServerDev };
```

Comment:
Right, the idea is a seamless upgrade path for people that are using the `react` package.

Maybe we don't need this after all though? It's definitely been discussed as something we'll do at various points, but I'd be open to not doing it.